sklearn/svm/tests/test_sparse.py

```diff
@@ -14,15 +14,15 @@
 
 # test sample 1
 X = np.array([[-2, -1], [-1, -1], [-1, -2], [1, 1], [1, 2], [2, 1]])
-X_sp = sparse.lil_matrix(X)
+X_sp = sparse.lil_matrix(X, dtype=np.int64)
```

Comment:
Thought this would fix it but did not. Will wait for travis to run so that we may all see the errors.